cmake/FDBComponents.cmake

```diff
@@ -20,41 +20,36 @@ endif()
 
 include(CheckSymbolExists)
 
-set(DISABLE_TLS OFF CACHE BOOL "Don't try to find OpenSSL and always build without TLS support")
 set(USE_WOLFSSL OFF CACHE BOOL "Build against WolfSSL instead of OpenSSL")
 set(USE_OPENSSL ON CACHE BOOL "Build against OpenSSL")
-if(DISABLE_TLS)
-  set(WITH_TLS OFF)
-else()
-  if(USE_WOLFSSL)
-    set(WOLFSSL_USE_STATIC_LIBS TRUE)
-    find_package(WolfSSL)
-    if(WOLFSSL_FOUND)
-      set(CMAKE_REQUIRED_INCLUDES ${WOLFSSL_INCLUDE_DIR})
-      set(WITH_TLS ON)
-      add_compile_options(-DHAVE_OPENSSL)
-      add_compile_options(-DHAVE_WOLFSSL)
-    else()
-      message(STATUS "WolfSSL was not found - Will compile without TLS Support")
-      message(STATUS "You can set WOLFSSL_ROOT_DIR to help cmake find it")
-      set(WITH_TLS OFF)
-    endif()
-  elseif(USE_OPENSSL)
-    set(OPENSSL_USE_STATIC_LIBS TRUE)
-    if(WIN32)
-      set(OPENSSL_MSVC_STATIC_RT ON)
-    endif()
-    find_package(OpenSSL)
-    if(OPENSSL_FOUND)
-      set(CMAKE_REQUIRED_INCLUDES ${OPENSSL_INCLUDE_DIR})
-      set(WITH_TLS ON)
-      add_compile_options(-DHAVE_OPENSSL)
-    else()
-      message(STATUS "OpenSSL was not found - Will compile without TLS Support")
-      message(STATUS "You can set OPENSSL_ROOT_DIR to help cmake find it")
-      set(WITH_TLS OFF)
-    endif()
+if(USE_WOLFSSL)
```

Comment:
Is anyone in the community relying on WolfSSL, do we post it as an update on the forum?

Reply:
I'd be shocked if anyone is relying in WolfSSL in such a short time @sfc-gh-ahusain. 

WolfSSL is useful for debugging TLS configuration issues as it can be made to be very verbose and could be made faster than OpenSSL if we did not use Wolf's OpenSSL compatibility layer. It can live in a branch. I would prefer to leave the FindWolfSSL.cmake file around like the FindLibreSSL.cmake file in case anyone needs to use it.

Reply:
I see, if it is useful for debugging we should probably leave it. Though we should print a warning that we don't test with it (CI only builds with OpenSSL IIRC -- so even the build might not be stable).

Reply:
I am planning to add a build in snowflake CI to make sure it builds successfully 🙂